Wishlist/wishlist.css

```diff
@@ -0,0 +1,31 @@
+@import url("/style.css");
+.card-container {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+}
+
+.cards {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 1rem;
```

Comment:
refer for above comment

Reply:
Thankyou for the suggestion. Will make the change